### DISCUSSION TITLE
Can`t use ZI

### DISCUSSION THREAD

**Discussion by oliveirarafa** [ajeetdsouza/zoxide#698] [Q&A] [answered]
I'm testing zoxide for the first time but can't make the zi command work. z works fine, but zi just give me blank, not even an error.

I installed zoxide, installed fzf, configured my .zshrc.

zoxide 0.9.3
fzf 0.46.1
zsh 5.9

.zshrc
```
...
eval "$(zoxide init --cmd cd zsh)"
```

**Comment by va9iff** [comment]
the reverse was a case for me: when using `--cmd cd` (in bash) I was no longer able to use my `cd` command. terminal just hangs up  and I could only `ctrl c` but `cdi` was fine. so I used `alias cd=z` and it works fine now (another alias for `cdi` and it works as well)

**Reply by ajeetdsouza** [reply]
Your zoxide is likely out of date. Did you install it with `apt`? https://github.com/ajeetdsouza/zoxide/issues/694

**Reply by va9iff** [reply]
yes I installed with `apt`

**Reply by va9iff** [reply]
also my `z webdev<space><tab>` doesn't bring the fuzzy window :d so I have to use `zi`

**Reply by ajeetdsouza** [reply]
Check #694, you need to use a more recent version of zoxide.

**Comment by ajeetdsouza** [comment]
@oliveirarafa is zoxide itself working correctly? Does `zoxide query -ls` return anything?

**Reply by oliveirarafa** [reply]
Yes, it returns a list of folders with their weights.

**Comment by AlphaCraft9658** [comment, marked as answer]
> I'm testing zoxide for the first time but can't make the zi command work. z works fine, but zi just give me blank, not even an error.
> 
> I installed zoxide, installed fzf, configured my .zshrc.
> 
> zoxide 0.9.3
> fzf 0.46.1
> zsh 5.9
> 
> .zshrc
> ```
> ...
> eval "$(zoxide init --cmd cd zsh)"
> ```

You told it to map to the cd command. Shouldn't you use cdi in this case?

**Reply by oliveirarafa** [reply]
Yeah, it works. Since I never used it before I followed a tutorial, there even with this cd mapping zi worked. Buts using cdi worked for me. Thank you, it was just me not knowing how to work with the tool.

**Reply by AlphaCraft9658** [reply]
> Yeah, it works. Since I never used it before I followed a tutorial, there even with this cd mapping zi worked. Buts using cdi worked for me. Thank you, it was just me not knowing how to work with the tool. 

Honestly, I also just started using it. :P
